docs/pages/config-plugins/introduction.mdx

```diff
@@ -1,59 +1,62 @@
 ---
-title: 'Config plugins: Introduction'
-description: An introduction to config plugins for the Expo project.
+title: Introduction to config plugins
+description: An introduction Expo config plugins.
 sidebar_title: Introduction
 ---
 
-import { Terminal } from '~/ui/components/Snippet';
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
 
-An automatic setup for adding a native module to your project is possible. Sometimes, a module requires a more complex setup. A config plugin can be used to automatically configure your native project for a module and reduce the complexity by avoiding interaction with the native project.
+import { BoxLink } from '~/ui/components/BoxLink';
+
+When using a [Continuous Native Generation (CNG)](/workflow/continuous-native-generation/) project, you might want to implement native project (**android** and **ios** directories) changes without directly interacting with the native project files. In such scenarios, you can use a config plugin to automatically configure your native project.
 
 ## What is a config plugin
 
-Config plugin is a system for extending the [app config](/workflow/configuration) and customizing the prebuild process for your app. They can be used to add native modules that aren't included, by default, or to add any native code that needs to be configured further.
+A config plugin is made up of individual plugin functions that are not already built into the [app config](/workflow/configuration). These functions are used to modify the native projects created during the [prebuild](/workflow/continuous-native-generation/#usage) process in CNG projects. Config plugins are written in JavaScript and are executed during the prebuild process.
```

Comment:
Not sure what "plugin functions" means here (I mean, I can visualize it, but it's hard to see where this fits in with "mods" below)

Reply:
or maybe if this is the introduction to plugin functions, let's start with what a plugin is without mentioning what's inside, and then say, "a plugin is typically made up of several plugin functions, which perform individual changes".